Updating to the new versions OCPP1.6 schemas

darander opened the pull request:
OCA has released new versions of the OCPP1.6 schemas together
with erratas.

This is the latest version.

Fixes #57 (adding Celsius with correct spelling) and more.

Signed-off-by: Anders Darander <anders@chargestorm.se>

darander commented:
Thanks for the comment.

I've updated the commit to omit the id lines.

OrangeTux commented on ocpp/v16/schemas/StopTransaction.json:
```diff
@@ -1,166 +1,175 @@
+                                        "A",
+                                        "V",
+                                        "K",
+                                        "Celcius",
```
It seems that `Celsius` is missing here.

darander replied:
That's right.
I'll bring it back to OCA, as I prefer to not touch the schemas here. (I think it's better to use the upstream schemas if possible).

OrangeTux replied:
Fair point.
I guess that the OCA fix the bug in a new V5 errate. We can implement the fix than after the errata has been released.

For now I am ready to merge the PR. Thanks for your help!

tropxy replied:
I also have contacted the OCA. It seems that they on purpose want the schemas to remain unchanged and that, for now, CS must adapt the enumerations and include both spellings and CP must correct the spelling.

OrangeTux commented:
For some reason the builds on CirceCi haven't been run. I will try to fix that. If tests pass I'm happy to merge!

darander commented:
I've added Celsius to StopTransaction.json, OCA is notified, but I rather not wait for them anyway.

I've also updated the commit message with info on when the new schema files were released and which modifications we've done to them.

OrangeTux commented:
@darander I fixed CircleCI's settings and now the test suite has ran. Unfortunately the build fails, can you fix it?

darander commented:
Fixed the build.

Thanks for getting the CircleCI working again!

OrangeTux commented:
Many thanks for your help!

darander commented:
Thanks for your help with my contribution!
